pkg/api/validation/schema.go

```diff
@@ -88,6 +88,9 @@ func (s *SwaggerSchema) ValidateBytes(data []byte) error {
 		return fmt.Errorf("kind not set")
 	}
 	allErrs := s.ValidateObject(obj, apiVersion.(string), "", apiVersion.(string)+"."+kind.(string))
+	if len(allErrs) > 0 {
+		glog.Warningf("found schema errors; if you choose to ignore them, turn validation off with --validate=false")
```

Comment:
Outputing this warning here will not respect the `io.Writer` that has been set for the command and it leaks warning level output from a utility function.  Maybe turn the message into a constant that can be referenced at the point of use in the commands?